api/src/auth/strategies/standard/validate.ts

```diff
@@ -26,7 +26,7 @@ export const StandardCredentials: TCredentials = {
   },
 
   fromRequest (request: Hapi.Request) {
-    return { ...request.auth.credentials.user, scope: request.auth.credentials.scope };
+    return Object.assign({ scope: request.auth.credentials.scope }, request.auth.credentials.user);
```

Comment:
Isn't the spread and `Object.assign` doing the same thing?

Reply:
Somehow TS loses typings if I use spread here